__tests__/test-utils.js

```diff
@@ -228,8 +228,23 @@ class TestUtils {
     });
   }
 
+  static startServer() {
+    return new Promise((resolve, reject) => {
+      const app = createApp();
+      server = app.listen(8088, err => {
+        if (err) {
+          reject(err);
```

Comment:
```suggestion
          return reject(err);
```
It's recommended to finish the function execution here, otherwise the supertest agent will also be instantiated and the promise will be also resolved, which might raise exceptions.

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-headless/pull/306/commits/652c02ab2bc917f8b2b6550bcdbbef883f1e60aa